docs/porting/display.md

```diff
@@ -69,6 +69,10 @@ LVGL might render the screen in multiple chunks and therefore call `flush_cb` mu
 
 ### Optional fields 
 There are some optional display driver data fields:
+- `physical_hor_res` horizontal resolution of the full / physical display in pixels. Only relevant when _not_ using fullscreen mode, otherwise set to -1.
```

Comment:
This comment suggests that the user needs to set it to `-1` is unused. I'd rather say: "...set it only if not using the full screen..."

Reply:
Ah, you're right, yes. I changed it.